src/terrain/clients/resource_usage_api.clj

```diff
@@ -0,0 +1,103 @@
+(ns terrain.clients.resource-usage-api
+  (:require [cemerick.url :as curl]
+            [clj-http.client :as http]
+            [terrain.util.config :as config]))
+
+(defn- resource-usage-api
+  ([components]
+   (resource-usage-api components {}))
+  ([components query]
+   (-> (apply curl/url (config/resource-usage-api-uri) components)
+       (assoc :query query)
+       str)))
+
+(defn current-cpu-hours-total
+  ([]
+   (-> (resource-usage-api ["admin" "cpu" "totals"])
+       (http/get {:as :json})
+       (:body)))
+  ([username]
+   (-> (resource-usage-api [username "cpu" "total"])
```

Comment:
I don't know if this can really come up, but I guess if `"admin"` is passed into this function as `username`, it'll behave like the admin version. I guess it might be worth excluding that case so the admin version is definitely only the 0-argument version (I figure just wrapping this side of the function in a `(when-not (= username "admin") )` would be sufficient, we don't need it to actually return a real value)

Reply:
(this would apply to anything that has the same structure too, I guess)

Reply:
Ooh, good point. I'll get a fix in. Thanks!